config/printer-eryone-thinker-series-v2.cfg

```diff
@@ -0,0 +1,225 @@
+#endstop_pin: probe:z_virtual_endstop
+#position_min: -2
+
+[extruder]
+step_pin: PC3
+dir_pin: !PL6
+enable_pin: !PA4
+microsteps: 16
+rotation_distance: 32
+nozzle_diameter: 0.400
+filament_diameter: 1.750
+heater_pin: PH6
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PF0
+control: pid
+pid_kp: 22.2
+pid_ki: 1.08
+pid_kd: 144
+min_temp: 0
+max_temp: 250
+max_extrude_only_distance: 300
+# Calibrate - see https://github.com/KevinOConnor/klipper/blob/master/docs/Pressure_Advance.md
+#pressure_advance: 0.0
+
+[fan]
+pin: PH5
+
+[heater_fan nozzle_cooling_fan]
+pin: PH3
+
+[heater_bed]
+heater_pin: PE5
+sensor_type: EPCOS 100K B57560G104F
+sensor_pin: PF2
+control: pid
+pid_kp: 100.0
+pid_ki: 5.023
+pid_kd: 305.4
+min_temp: 0
+max_temp: 100
+
+[mcu]
+serial: /dev/serial/by-id/usb-1a86_USB2.0-Serial-if00-port0
+
+[printer]
+kinematics: cartesian
+max_velocity: 300
+max_accel: 1500
+max_z_velocity: 5
+max_z_accel: 150
+
+[pause_resume]
+
+[virtual_sdcard]
+path: ~/.octoprint/uploads/
+
+[respond]
+
+######################################################################
+##  Filament runout sensor settings
+##  Uncomment the following if using BL Touch and adding runout sensor
+######################################################################
+
+#[filament_switch_sensor filament_runout_sensor]
+#runout_gcode: M600
+#switch_pin: !PB4
+
+[gcode_macro M600]
+gcode:
+    PAUSE
+    CHANGE_FILAMENT_START
+
+[gcode_macro CHANGE_FILAMENT_START]
+gcode:
+    SAVE_GCODE_STATE NAME=M600_state
+    G91
+    G1 E-.8 F2700
+    G1 Z10
+    G90
+    G1 X50 Y0 F3000
+    G91
+    G1 E-5 F300
+
+[gcode_macro CHANGE_FILAMENT_END]
+gcode:
+    G91
+    RESTORE_GCODE_STATE NAME=M600_state
+    RESUME
+
+######################################################################
+##  Probing/Mesh Leveling settings
+##  Uncomment the following if using BL Touch.
+##  Please make sure you set the correct values for your offsets.
+######################################################################
+
+#[bltouch]
+#sensor_pin: ^PA0  # Pull-up (^ symbol) needed in open drain mode
+#control_pin: PG5
+#x_offset: 30
+#y_offset: 0
+#z_offset: 0
+#samples: 2
+#pin_move_time: 0.8
+
+#[bed_mesh]
+#mesh_min: 10,10
+#mesh_max: 290,290
+
+#[safe_z_home]
+#home_xy_position: 120,150
+#speed: 80.0
+#z_hop: 10.0
+#z_hop_speed: 10.0
+
+[gcode_macro G29]
+gcode:
+    BED_MESH_CALIBRATE
+    BED_MESH_PROFILE SAVE=p1
+
+[gcode_macro M420]
+gcode:
+    BED_MESH_OUTPUT
+
+######################################################################
+# GCode Overrides
+######################################################################
+
+[gcode_macro M500]
+gcode:
+    SAVE_CONFIG
+
+[gcode_macro M300]
+default_parameter_S: 1000
+# Use a default 1kHz tone if S is omitted.
+default_parameter_P: 100
+# Use a 10ms duration is P is omitted.
+gcode:
+    SET_PIN PIN=beeper VALUE={S}
+    G4 P{P}
+    SET_PIN PIN=beeper VALUE=0
+
+[gcode_macro M205]
+gcode:
```

Comment:
Similar comment.

Reply:
Can I have them commented out for user?  Again the community feedback is when they try Klipper their is a lot of functionality that doesn't work out of the box for them.  Ideally these would be part of the core.